ui/shared/Revision.jsx

```diff
@@ -94,13 +126,34 @@ export class Revision extends React.PureComponent {
         </span>
         <AuthorInitials title={`${name}: ${email}`} author={name} />
         <span
-          title={comment}
+          ref={this.spanRef}
```

Comment:
Do we still need this `ref` and `ready` code?  If the bug summaries haven't loaded yet, then the tooltip can just fall-back to the default of showing the commit comment.  Maybe we can take this out?

Reply:
The ref/ready was because issues displaying `ToolTip`. Since we're just using `UncontrolledToolTip` now we don't need it anymore. Took it out!